topics/synthetic-biology/tutorials/basic_assembly_analysis/workflows/Genetic_Design_BASIC_Assembly.ga

```diff
@@ -0,0 +1,413 @@
+{
+    "a_galaxy_workflow": "true",
```

Comment:
```suggestion
    "a_galaxy_workflow": "true",
    "tags": ["synthetic-biology"],
```

Reply:
oh wait, I saw you already did that, wonder why the test is still failing, will look into it, disregard this suggestion